ui/components/app/transaction-decoding/transaction-decoding.component.js

```diff
@@ -99,6 +108,7 @@ export default function TransactionDecoding({ to = '', inputData: data = '' }) {
         }
       }
     })();
+    return () => setMounted(false);
   }, [t, from, to, network, data]);
 
   // ***********************************************************
```

Comment:
This file's change are not directly related to task,  but were needed to remove warnings from new test cases added.